### ISSUE TITLE
My Cleverbot logic suddenly stopped working and now gives me this error!

### ISSUE THREAD

**Issue by Beefywhale** [folz/cleverbot.py#39]
```Python
  File "WeenieBot.py", line 30, in cleverbot_logic
    answer = str(cb.ask(question))
  File "/usr/local/lib/python3.5/dist-packages/cleverbot/cleverbot.py", line 104, in ask
    parsed = self._parse(resp.text)
  File "/usr/local/lib/python3.5/dist-packages/cleverbot/cleverbot.py", line 156, in _parse
    if parsed[0][1] == 'DENIED':
IndexError: list index out of range```

**Comment by folz** [comment]
Hey there, your problem should be fixed by v1.1.1 on pypi. Please update and let me know if that works for you.

**Comment by Beefywhale** [comment]
Thanks works perfect now! :smile:
